packages/renderer/src/renderer.ts

```diff
@@ -82,30 +83,55 @@ export class Renderer implements IAgentPlugin {
 
       // TODO: There's an issue with W3 availability causing the fetching of some W3 context files to fail. Since we know the exact location of the template we can bypass the JSONLD expansion. This is a temporary workaround.
 
-      const render = args.credential.render;
+      const render = args.credential.render || args.credential.renderMethod;
```

Comment:
Previous, we had hotfix for the performance issue. So please follow the TODO to handle it right